BayPass/APIs/Here.swift

```diff
@@ -0,0 +1,276 @@
+//
+//  Here.swift
+//  BayPass
+//
+//  Created by Ayesha Khan on 2/27/19.
+//  Copyright © 2019 Tim Roesner. All rights reserved.
+//
+
+import Alamofire
+import Foundation
+import MapKit
+
+class Here {
```

Comment:
Did you decide on a data structure that fits our use case? I'd store it in a Singleton and initialize it with all stations and lines from the AppDelegate